fast64_internal/f3d/f3d_enums.py

```diff
@@ -165,20 +165,20 @@
 ]
 
 enumCombKey = [
-    ("G_CK_NONE", "None", "None"),
-    ("G_CK_KEY", "Key", "Key"),
+    ("G_CK_NONE", "None", "Disables chroma key."),
+    ("G_CK_KEY", "Key", "Enables chroma key."),
 ]
 
 enumTextConv = [
-    ("G_TC_CONV", "Convert", "Convert"),
-    ("G_TC_FILTCONV", "Filter And Convert", "Filter And Convert"),
-    ("G_TC_FILT", "Filter", "Filter"),
+    ("G_TC_CONV", "Convert", "Convert, used for YUV to RGB conversion."),
+    ("G_TC_FILTCONV", "Filter And Convert", "Filter And Convert, used for YUV to RGB conversion."),
```

Comment:
I'm pretty sure this is filter in the first cycle and convert in the second cycle, but you might want to double check.

Reply:
This is correct, I have verified it using the RDP command summary.